flowzone.yml

```diff
@@ -999,10 +999,14 @@ jobs:
 
       - name: Install dependencies
         run: |
-          if [ -e package-lock.json ] || [ -e npm-shrinkwrap.json ]; then
-            npm ci
-          else
-            npm i
+          runner_os="$(echo "${RUNNER_OS}" | tr '[:upper:]' '[:lower:]')"
+          os="$(cat < package.json | yq e .os[])"
```

Comment:
```suggestion
          exists="$(jq --arg os "$runner_os" '.os | index($os)' package.json)"
```

Reply:
it only makes sense to compare runner_os with os key for value `linux`. That's the only convergence, between the two sets of values. There are more verbose way of checking, or we can just limit the npm steps to run `if: runner_os == 'Linux'`.